packages/api/src/handlers/queues.ts

```diff
@@ -29,7 +29,7 @@ const formatJob = (job: QueueJob, queue: BaseAdapter): AppJob => {
     stacktrace,
     opts: jobProps.opts,
     data: queue.format('data', jobProps.data),
-    name: queue.format('name', jobProps, jobProps.name),
+    name: queue.format('name', jobProps, jobProps.name || ''),
```

Comment:
It is better to make a default value outside of the format it self

```
queue.format('name', jobProps, jobProps.name) || '',
```

Reply:
When the `jobProps.name` is undefined, the `format()` will return `jobProps`. So, we can't make a default value outside of the `format`.

Reply:
I think this might be good too:

```typescript
queue.format('name', jobProps.name, '')
```